app/import.go

```diff
@@ -38,44 +38,74 @@ func stopOnError(err LineImportWorkerError) bool {
 	}
 }
 
-func rewriteAttachmentPaths(files *[]AttachmentImportData, basePath string) {
+func processAttachmentPaths(files *[]AttachmentImportData, basePath string, filesMap map[string]*zip.File) error {
 	if files == nil {
-		return
+		return nil
 	}
-	for _, f := range *files {
+	var ok bool
+	for i, f := range *files {
 		if f.Path != nil {
-			*f.Path = filepath.Join(basePath, *f.Path)
+			path := filepath.Join(basePath, *f.Path)
+			*f.Path = path
+			if len(filesMap) > 0 {
+				if (*files)[i].Data, ok = filesMap[path]; !ok {
+					return fmt.Errorf("attachment %q not found in map", path)
+				}
+			}
 		}
 	}
+
+	return nil
 }
 
-func rewriteFilePaths(line *LineImportData, basePath string) {
+func processAttachments(line *LineImportData, basePath string, filesMap map[string]*zip.File) error {
+	var ok bool
 	switch line.Type {
 	case "post", "direct_post":
 		var replies []ReplyImportData
 		if line.Type == "direct_post" {
-			rewriteAttachmentPaths(line.DirectPost.Attachments, basePath)
+			if err := processAttachmentPaths(line.DirectPost.Attachments, basePath, filesMap); err != nil {
+				return err
```

Comment:
Will these returns cause the import to stop? If so, we want to log the error in some way and continue instead of returning